app.rb

```diff
@@ -0,0 +1,89 @@
+require './person'
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'All books'
+    @books.each { |book| puts book }
+  end
```

Comment:
- [ ] Good job on your logic 👍🏽 . Kindly take note that your list of books should return the actual books  and not hash to the book
![image](https://github.com/eudondian/opp-school-library/assets/63473584/b4b51a96-347e-4c6f-a246-f975425b6881)


your list all books logic can look like this 👇🏽 
  ```
# *List all books
  def list_all_books
    @books.each do |book|
      puts "Title: \"#{book.title}\", Author: #{book.author}"
    end
  end

```